joint_names: [r_shoulder_pan_joint, r_shoulder_lift_joint, r_upper_arm_roll_joint, r_elbow_flex_joint, r_forearm_roll_joint, r_wrist_flex_joint, r_wrist_roll_joint]

r_shoulder_pan_joint:
  joint: r_shoulder_pan_joint
  goal_reached_threshold: 0.01
  pid:
    p: 60.0
    d: 5.0
    i: 10.0
    i_clamp: 4.0

r_shoulder_lift_joint:
  joint: r_shoulder_lift_joint
  goal_reached_threshold: 0.01
  pid:
    p: 60.0
    d: 5.0
    i: 10.0
    i_clamp: 4.0

r_upper_arm_roll_joint:
  joint: r_upper_arm_roll_joint
  goal_reached_threshold: 0.01
  pid:
    p: 25.0
    d: 2.0
    i: 1.0
    i_clamp: 1.0

r_elbow_flex_joint:
  joint: r_elbow_flex_joint
  goal_reached_threshold: 0.01
  pid:
    p: 100.0
    d: 10.0
    i: 20.0
    i_clamp: 2.0

r_forearm_roll_joint:
  joint: r_forearm_roll_joint
  goal_reached_threshold: 0.01
  pid:
    p: 300.0
    d: 20.0
    i: 0.0
    i_clamp: 0.0

r_wrist_flex_joint:
  joint: r_wrist_flex_joint
  goal_reached_threshold: 0.01
  pid:
    p: 200.0
    i: 8.0
    d: 0.0
    i_clamp: 0.0

r_wrist_roll_joint:
  joint: r_wrist_roll_joint
  goal_reached_threshold: 0.01
  pid:
    p: 200.0
    d: 8.0
    i: 0.0
    i_clamp: 0.0
